pkg/jsonlog/jsonlogbytes.go

```diff
@@ -3,16 +3,15 @@ package jsonlog
 import (
 	"bytes"
 	"encoding/json"
+	"time"
 	"unicode/utf8"
 )
 
-// JSONLogs is based on JSONLog.
-// It allows marshalling JSONLog from Log as []byte
-// and an already marshalled Created timestamp.
+// JSONLogs marshals encoded JSONLog objects
 type JSONLogs struct {
-	Log     []byte `json:"log,omitempty"`
-	Stream  string `json:"stream,omitempty"`
-	Created string `json:"time"`
+	Log     []byte    `json:"log,omitempty"`
+	Stream  string    `json:"stream,omitempty"`
+	Created time.Time `json:"time"`
```

Comment:
We are using that one on the client side right ? This might break a bunch of user of this `pkg` right ?

Reply:
This struct is not used by the cli. The cli is only using a constant, and is made forward compatible here: https://github.com/docker/cli/pull/551

It might break users of this `pkg`, but that is unfortunately necessary. These packages were not maintained properly and developed really bad interfaces which need to be fixed.  We're going to be breaking everyone with #32989 anyway, and I think this can be seen as cleanup work necessary for #32989.

The godoc (https://godoc.org/github.com/moby/moby/pkg/jsonlog) doesn't show any consumers of this package. I'm not sure how reliable that is, but it seems to work for other packages.